sdk/Lusid.Sdk.Tests/Utilities/DemoInstrumentBase.cs

```diff
@@ -119,12 +119,16 @@ internal void CallLusidGetPortfolioCashFlowsEndpoint(LusidInstrument instrument,
                 // SimpleStatic pricing is lookup pricing. As such, we upsert a quote.
                 // Note that inside CreatePortfolioAndInstrument, the method TestDataUtilities.BuildInstrumentUpsertRequest books the instrument using "ClientInternal".
                 // Hence upsert a quote using ClientInternal as the instrumentIdType.
-                var quoteRequest = TestDataUtilities.BuildQuoteRequest(
+                var quoteRequest = new Dictionary<string, UpsertQuoteRequest>();
+                TestDataUtilities.BuildQuoteRequest(
+                    quoteRequest,
+                    "req",
```

Comment:
what is "req"?

Reply:
I'm not sure.... this was used previously - I have just refactored the code but not changed this parameter. Can put a clearer name.